xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1030 @@
+     download all the images. This approach vastly simplifies the task of copying
+     charts and their related images.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.requirements">
+   <title>Requirements</title>
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.machines">
+    <title>Mirror Servers</title>
+    <note>
+     <title>Shared Mirror Server</title>
+     <para>
+      If you have multiple clusters or a very large amount of nodes accessing
+      these mirrors, you should increase the sizing of CPU/RAM.
+     </para>
+     <para>
+      Storage sizing depends on your intended update frequency and data
+      retention model. If you want to keep snapshots or images of repository
+      states at various points, you must increase storage size accordingly.
+     </para>
+    </note>
+    <para>
+     You will need to provide and maintain at least two machines in addition to
+     your &productname; cluster. These mirror servers will reside on the
+     external part of your network and the internal (airgapped) network
+     respectively.
+    </para>
+    <para>
+     <link xlink:href="https://www.suse.com/documentation/sles-15/book_quickstarts/data/art_sle_installquick.html">Set
+     up two &sls; 15 machines</link> one on the internal network and one on
+     the airgapped network.
+    </para>
+    <para>
+     Make sure you have
+     <link xlink:href="https://www.suse.com/documentation/sles-15/book_sles_docker/data/preparation.html">enabled
+     the <filename>Containers</filename> module</link> on each server.
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>External</term>
+      <listitem>
+       <para>
+        This machine will host the <literal>&rmtool;</literal> for RPM packages
+        and the <literal>Docker registry</literal> for container
+        images.
+       </para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          <literal>1</literal> Host machines for the mirror servers.
+         </para>
+         <itemizedlist>
+          <listitem>
+           <para>
+            SLES 15
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            2 (v)CPU
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            8 GB RAM
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            500 GB Storage
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </itemizedlist>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Internal (Airgapped)</term>
+      <listitem>
+       <para>
+        This machine will host the <literal>&rmtool;</literal> for RPM packages,
+        and <literal>Docker registry</literal> for container images
+        as well as a webserver hosting the <literal>Helm chart repository</literal>
+        files.
+       </para>
```

Comment:
if it's a separate server the procedure will get slightly more complicated because they have to somehow transfer the data from the secure storage to that server, we're pointing to the SLES docs that explain how to configure a webserver

Is that enough?

Reply:
Who is taking care of this? I explain later that they should configure the helm repo server to be available on a different webserver root. Isn't that enough? I could just make this mandatory?
RMT server would use the normal endpoint and helm chart server at /charts or is there a port conflict? I couldn't find any network requirements for RMT or SLE15.

Reply:
@chentex can you take a look at a deployed RMT machine? I think they are using nginx as web server. It would be just a matter of adding a new vhost or whatever is less intrusive

Reply:
OK, will do that today